app/views/layouts/mailer.html.erb

```diff
@@ -1010,6 +1010,8 @@ h1 {
               <p>Have fun and take care,</p>
 
               <p><a href="https://ruby.org.au">Ruby Australia</a></p>
+              <p>P.S. Australian legislation requires that we inform all members of Ruby Australia about our AGM and SGMs. If you no longer wish to receive these notifications, you can resign your membership from Ruby Australia <a href="https://ruby.org.au/my/details">here<a/></p>
+              <p>You are receiving this email as a registered member of Ruby AU</p>
```

Comment:
Thanks so much for this, looking at it now, can we make the minor change it to of moving the last sentence to the start of the PS. (and changing the Ruby AU to Ruby Australia) 
`P.S. You are receiving this email as a registered member of Ruby Australia. Australian legislation requires... etc`
and move it into the div starting on 1031 with 1 line break in between the PS and the address so that they're both in the footer.

Reply:
@quintrino I've added the changes to the next commit. Please let me know if I missed any.